app/services/activities/import_from_csv.rb

```diff
@@ -284,19 +285,30 @@ def to_h
       end
 
       def convert_to_attributes
-        attributes = FIELDS.each_with_object({}) { |(attr_name, column_name), attrs|
+        attributes = fields.each_with_object({}) { |(attr_name, column_name), attrs|
           attrs[attr_name] = convert_to_attribute(attr_name, @row[column_name]) if field_should_be_converted?(column_name)
         }
 
-        attributes[:geography] = infer_geography(attributes)
-        attributes[:requires_additional_benefitting_countries] = (@row["Recipient Country"] && @row["Intended Beneficiaries"]).present?
-        attributes[:recipient_region] ||= inferred_region
-        attributes[:call_present] = (@row["Call open date"] && @row["Call close date"]).present?
-        attributes[:sector_category] = get_sector_category(attributes[:sector])
+        if @method == :create
```

Comment:
What happens if there is a change in the attributes that determine geography during update? Isn't the existing data going to be mismatched with the new data from the upload?

Reply:
Good shout 👍